app/pods/flattening-array-with-cps/controller.js

```diff
@@ -9,11 +9,13 @@ export default Controller.extend({
   model: null, // Populated from route
   owners: readOnly('model'),
 
+  allPetsOwner: computed('owners', function() { 
```

Comment:
Have you checked that this solution actually works in the browser?

Here you are missing the `.[]` part of the depencency key. I think...